docs/stdlib/generic.rst

```diff
@@ -134,11 +146,11 @@ Generic
 
 .. eql:operator:: lt: anytype < anytype -> bool
 
-    Less than operator.
+    Performs a less than value comparison with two valeues of
+    :eql:type:`anytype`.
 
-    Return ``true`` if the value of the left expression is less than
-    the value of the right expression. In EdgeQL any values can be
-    compared to each other as long as they are of the same type:
+    This results in a :eql:type:`bool` of ``true`` if the left operand
```

Comment:
With part of this now encompassed in the one-liner, this can probably mirror the others.

```suggestion
    This results in a :eql:type:`bool`:
```